frontend/src/router.ts

```diff
@@ -15,54 +15,34 @@ export const router = new Router({
   routes: [
     { path: "*", redirect: "/" },
     {
-      path: "/login",
-      meta: { auth: { guestOnly: true } },
+      path: "/",
+      meta: { auth: { requireAuth: true } },
```

Comment:
Do we really need to apply this constraint to the root route? I feel it's redundant because the nested routes which require authentication, they are explicitly requiring a role.

Consider removing this constraint and redundant `requireAuth: false` for anonymous nested routes.